src/opera/commands/deploy.py

```diff
@@ -95,3 +132,18 @@ def deploy(service_template: str, inputs: typing.Optional[dict], storage: Storag
     template = ast.get_template(inputs)
     topology = template.instantiate(storage)
     topology.deploy(num_workers)
+
+
+def prompt_yes_no_question():
```

Comment:
This really seems like something that should be in a separate file, maybe `opera/commands/__init__.py`? So you'd `from opera.commands import prompt_yes_no_question`

But then you'd run into circular imports, so maybe a utils file?

Reply:
I see. For now I've created utils.py file where prompt function resides.